assets/package.json

```diff
@@ -71,6 +73,7 @@
     "prettier": "^2.7.1",
     "react-test-renderer": "^18.2.0",
     "sass-loader": "^13.0.2",
+    "screenfull": "^6.0.2",
```

Comment:
suggestion (non-blocking): Maybe worth adding an in-line comment as to why this is a dev dependency?

Reply:
I tried, but unfortunately it's a JSON file so it doesn't like comments in the syntax.